types/openfin/_v2/api/platform/platform-provider.d.ts

```diff
@@ -3,64 +3,54 @@ import { OverrideCallback } from '../../shapes/Platform';
 export default function initConstructor(environment: Environment): (userOverrideFn: OverrideCallback<any>) => Promise<void>;
 /**
  * ApplySnapshotOptions interface
- * @typedef { object } ApplySnapshotOptions
```

Comment:
all these typedefs should be deleted, or if they exist as real type aliases elsewhere, moved there

Reply:
I ended up reverting this back to `master`. +1 to fixing it up, but I'm running low on time budget...